### PR TITLE
Update recipe to nvcomp 3.0.4.

### PR DESCRIPTION
**Pull request by bdice**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bdice** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% set version = "3.0.2" %}
+{% set version = "3.0.4" %}
```
Comment (jakirkham) [inline recipe/meta.yaml:1]: Do we want to publish 3.0.3 first or would we rather skip that one?

**Review by jakirkham — COMMENTED** [review]
Thanks Bradley! 🙏

Had one question below

File: recipe/meta.yaml (hunk 1 above)
Comment (bdice) [reply recipe/meta.yaml:1]: Skip. It has a bug. The only libraries depending on nvcomp today are within RAPIDS, I believe, and RAPIDS won’t be able to use 3.0.3.

**Review by jakirkham — APPROVED** [review]

**Pull request merged**